tests/transformers/test_generate_mask.py

```
@@ -0,0 +1,46 @@
+        [
+            [True, False, False, False, False],
+            [True, True, False, False, False],
+            [True, True, True, False, False],
+            [True, True, True, True, False],
+            [True, True, True, True, True],
+        ]
+    )
+    assert torch.allclose(mask, expected_mask), f"Expected mask: {expected_mask}, but got: {mask}"
+
+
+def test_make_tgt_mask():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_make_tgt_mask`

Add return type annotation: `None`

(ANN201)